src/support/utils/is.nullish.ts

```
@@ -0,0 +1,14 @@
+export function isNullish<T>(value: T): value is null {
+  return (
+    value === null ||
+    value === "" ||
+    value === undefined ||
+    value === "null" ||
+    value === "undefined" ||
+    (typeof value === "number" && isNaN(value))
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)